pom.xml

```diff
@@ -91,7 +91,7 @@
 
         <dep.airlift.version>247-SNAPSHOT</dep.airlift.version>
         <dep.bouncycastle.version>1.78</dep.bouncycastle.version>
-        <dep.bytebuddy.version>1.14.13</dep.bytebuddy.version>
+        <dep.bytebuddy.version>1.14.14</dep.bytebuddy.version>
```

Comment:
if Trino maintains its own byte buddy version, why would we move this version number in https://github.com/airlift/airbase/pull/408 to be controlled by airbase?

Reply:
It's shared between airlift and Trino - hence it should be defined in airbase